src/explore-education-statistics-admin/src/pages/release/components/ReleaseStatusChecklist.tsx

```diff
@@ -90,6 +90,22 @@ const ReleaseStatusChecklist = ({ checklist, release }: Props) => {
               releaseRouteParams,
             ),
           };
+        case 'EmptyContentSectionExists':
+          return {
+            message: 'Release content sections should not be empty',
```

Comment:
Would suggest re-wording to: 'Release content should not contain any empty sections' . This is a little more consistent with the other message and also sounds better for both plural and singular cases.